clients/admin-ui/src/features/configure-consent/ConsentManagementModal.tsx

```diff
@@ -0,0 +1,173 @@
+/* eslint-disable react/no-array-index-key */
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Button,
+  Flex,
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Spacer,
+  Spinner,
+  useDisclosure,
+} from "@fidesui/react";
+import { FieldArray, Form, Formik } from "formik";
+
+import {
+  CustomCreatableSelect,
+  CustomTextInput,
+} from "~/features/common/form/inputs";
+import { useGetSystemPurposeSummaryQuery } from "~/features/plus/plus.slice";
+import { SystemPurposeSummary } from "~/types/api";
+
+export const useConsentManagementModal = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  return { isOpen, onOpen, onClose };
+};
+
+type Props = {
+  isOpen: boolean;
+  onClose: () => void;
+  fidesKey: string;
+};
+
+type FormValues = SystemPurposeSummary;
+
+export const ConsentManagementModal = ({
+  isOpen,
+  onClose,
+  fidesKey,
+}: Props) => {
+  const { data: systemPurposeSummary, isLoading } =
+    useGetSystemPurposeSummaryQuery(fidesKey);
+
+  return (
+    <Modal
+      isOpen={isOpen}
+      onClose={onClose}
+      size="xxl"
+      returnFocusOnClose={false}
+      isCentered
+    >
+      <ModalOverlay />
+      <ModalContent maxWidth="800px">
+        <ModalHeader>Vendor</ModalHeader>
+        <ModalBody>
+          {isLoading ? (
+            <Flex
+              width="100%"
+              height="324px"
+              alignItems="center"
+              justifyContent="center"
+            >
+              <Spinner />
+            </Flex>
+          ) : (
+            <Formik<FormValues>
+              initialValues={
+                systemPurposeSummary as unknown as SystemPurposeSummary
+              }
+              enableReinitialize
+              onSubmit={() => {}}
+            >
+              {({ values }) => (
+                <Form>
+                  <Box mb={6}>
+                    <CustomTextInput
+                      label="Vendor Name"
+                      variant="stacked"
+                      name="name"
+                      disabled
+                    />
+                  </Box>
+                  <FieldArray
```

Comment:
I know it's not in the designs but it'd be helpful to add a field label to make it clear what we're displaying.
```suggestion
                  {!_.isEmpty(values.purposes) && (
                    <Label fontSize="xs">Purposes</Label>
                  )}
                  <FieldArray
```